rusty_pizza_server/src/order_model/order.rs

```diff
@@ -2,11 +2,49 @@ use crate::order_model::meal::{Meal, MealFactory};
 use crate::order_model::meals::Meals;
 use crate::order_model::user::User;
 use crate::util::money::Money;
-use std::collections::HashMap;
+use std::collections::{HashMap, HashSet};
 use std::error;
 use std::fmt;
 use std::rc::Rc;
 
+#[derive(Debug, PartialEq)]
+/// Not all user who take part in this Order has paid enough. The Error has two values:
```

Comment:
```suggestion
/// Not all users who take part in this Order have paid enough. The Error has two values:
```
or
```suggestion
/// Not every user taking part in this Order has paid enough. The Error has two values:
```

Reply:
I would also drop everything after the first sentence, the individual description should be enough.

Reply:
Done